tools/@aws-cdk/cdk-build-tools/lib/package-info.ts

```diff
@@ -142,14 +142,14 @@ export interface CDKBuildOptions {
   };
 
   /**
-   * An optional command (formatted as a list of strings) to run before building
+   * Optional commands (formatted as a list of strings, which will be joined together with the && operator) to run before building
```

Comment:
I did not change the logic for the pre and post build steps, but it was inaccurately described here, so I updated the comments.